src/components/reference/_cli-help-output.md

```diff
@@ -15,7 +15,8 @@ Commands:
   install-semgrep-pro  Install the Semgrep Pro Engine
   login                Obtain and save credentials for semgrep.dev
   logout               Remove locally stored credentials to semgrep.dev
-  lsp                  [EXPERIMENTAL] Start the Semgrep LSP server
```

Comment:
@ajbt200128 Hi Austin, we build the CLI output through a github workflow action that pipes the from the semgrep repo into the docs. Just an FYI, this might be overwritten (by the same content you put in :D ) so you don't have to be mindful and update this help output every time the Semgrep binary updates its --help output.